src/labelformat/formats/yolov8.py

```diff
@@ -50,28 +50,75 @@ def __init__(self, input_file: Path, input_split: str) -> None:
             )
 
     def get_categories(self) -> Iterable[Category]:
-        for category_id, category_name in self._config_data["names"].items():
-            yield Category(
-                id=int(category_id),
-                name=category_name,
+        """Get categories from YOLOv8 config file.
+
+        Supports two formats for the 'names' field:
+        - dictionary mapping id to name
+          Example:
+          names:
+            0: person
+            1: dog
+            2: cat
+        - list of names where index is the id
+          Example:
+          names:
+            - person
+            - dog
+            - cat
+          or
+          names: [person, dog, cat]
+        Returns:
+            Iterator of Category objects.
+        """
+        names = self._config_data["names"]
+        if isinstance(names, dict):
+            # Original YOLOv8 format: dictionary mapping id to name
+            for category_id, category_name in names.items():
+                yield Category(
+                    id=int(category_id),
+                    name=category_name,
+                )
+        elif isinstance(names, list):
+            # Alternative YOLO format: list of names where index is the id
+            for category_id, category_name in enumerate(names):
+                yield Category(
+                    id=category_id,
+                    name=category_name,
+                )
+        else:
+            raise ValueError(
+                f"Invalid 'names' format in config file '{self._config_file}'. "
+                "Expected dictionary or list, got {type(names)}"
             )
 
     def get_images(self) -> Iterable[Image]:
         yield from utils.get_images_from_folder(folder=self._images_dir())
 
     def _root_dir(self) -> Path:
-        return self._config_file.parent / str(self._config_data["path"])
+        if "path" in self._config_data:
+            return self._config_file.parent / str(self._config_data["path"])
+        return self._config_file.parent
```

Comment:
Can you add a comment?
```suggestion
        # Roboflow format assumes the data in the config file's parent.
        return self._config_file.parent
```

Also, are you sure this is correct? If you look at
```
train: ../train/images
val: ../valid/images

nc: 3
names: ['head', 'helmet', 'person']
```
shouldn't it be
```
train: train/images
val: valid/images

nc: 3
names: ['head', 'helmet', 'person']
```
?

Reply:
I added the comment. Regarding the `../train/images/` I tested it on a few roboflow datasets and the root folder is the folder containing the yaml file.